packages/hardhat-plugin/src/index.ts

```diff
@@ -54,18 +56,22 @@ extendEnvironment((hre) => {
   });
 });
 
-task("deploy")
-  .addPositionalParam("moduleNameOrPath")
+ignitionScope
+  .task("deploy")
+  .addPositionalParam(
+    "moduleNameOrPath",
+    "The name of the module file within the Ignition modules directory, or a path to the module file"
+  )
   .addOptionalParam(
     "parameters",
-    "A JSON object as a string, of the module parameters, or a relative path to a JSON file"
+    "A relative path to a JSON file to use for the module parameters,"
   )
   .addOptionalParam("id", "set the deployment id")
-  .addFlag("force", "restart the deployment ignoring previous history")
   .addFlag(
```

Comment:
I think we should remove this flag for now. It's unclear if it adds value when compared to the other UI. If we get complains about the normal UI, we can consider reintroducing it.

To be clear, I'd just remove the flag